pages/events.tsx

```diff
@@ -0,0 +1,11 @@
+import { NextPage } from 'next';
+
+const events: NextPage = () => {
```

Comment:
Steht in den Docs dass man den Component hier mit lowercase definieren soll? Weil wenn nicht dann bitte Components immer mit Uppercase starten.